Tools/scripts/summarize_stats.py

```diff
@@ -377,8 +552,7 @@ def emit_pair_counts(opcode_stats, total):
                     succ_rows
                 )
 
-def main():
-    stats = gather_stats()
+def output_single_stats(stats):
```

Comment:
Are you using the "emit_" and "output_" prefixes interchangeably, or is there a difference?

Reply:
It's keeping the naming from the original code (which is @markshannon's), which has `output_stats` as a top-level function (which I split into three).  I guess the difference is that `output_` is these top-level functions, whereas each of the `emit_` functions emits a single section.  But we certainly could use `emit_` everywhere.